biogrid/libs/biogrid-simulator/src/lib/bioenergy-source/bioenergy-source-class.ts

```diff
@@ -6,21 +6,24 @@ import {
   ItemPosition,
   Distance,
 } from '@biogrid/grid-simulator';
+import { GRID_ITEM_NAMES, SOLAR_PANEL } from '../config';
```

Comment:
I would look over this again after merging with master, this file name was changed in the last merged PR, now there is a solar panel file as well as a bioenergy source abstract class.

Reply:
which merge are you referring to? one in master or in this PR